pkg/api/kubegraph/edges.go

```diff
@@ -242,6 +243,8 @@ func AddHPAScaleRefEdges(g osgraph.Graph) {
 			syntheticNode = kubegraph.FindOrCreateSyntheticReplicationControllerNode(g, &kapi.ReplicationController{ObjectMeta: syntheticMeta})
 		case appsapi.IsResourceOrLegacy("deploymentconfigs", r):
 			syntheticNode = appsgraph.FindOrCreateSyntheticDeploymentConfigNode(g, &appsapi.DeploymentConfig{ObjectMeta: syntheticMeta})
+		case r == kapisext.Resource("deployments"):
```

Comment:
@tnozicka do we need to handle apps.Resource("deployments) here as well?

Reply:
It's being handled in the previous case.

Reply:
i'm talking about kapps

Reply:
This is the internal representation. Depoyment's internal are extensions, still.